--- ./wrapper/Makefile.org	Thu Jul  4 12:52:16 2002
+++ ./wrapper/Makefile	Fri May  9 14:45:03 2003
@@ -42,13 +42,16 @@
 endif
 
 install:
+	if [ ! -d $(DESTDIR)$(OH323WRAPLIBDIR) ]; then
+		$(INSTALL) -d $(DESTDIR)$(OH323WRAPLIBDIR)
+	fi
 ifeq ($(WRAPSTAT),1)
-	$(INSTALL) liboh323wrapd.a $(OH323WRAPLIBDIR)
+	$(INSTALL) liboh323wrapd.a $(DESTDIR)$(OH323WRAPLIBDIR)
 else
-	$(INSTALL) liboh323wrap.so $(OH323WRAPLIBDIR)
-	cd $(OH323WRAPLIBDIR); rm -f ./liboh323wrap.so.1; rm -f ./liboh323wrap.so.1.1
-	cd $(OH323WRAPLIBDIR); ln -s liboh323wrap.so liboh323wrap.so.1
-	cd $(OH323WRAPLIBDIR); ln -s liboh323wrap.so liboh323wrap.so.1.1
+	$(INSTALL) liboh323wrap.so $(DESTDIR)$(OH323WRAPLIBDIR)
+	cd $(DESTDIR)$(OH323WRAPLIBDIR); rm -f ./liboh323wrap.so.1; rm -f ./liboh323wrap.so.1.1
+	cd $(DESTDIR)$(OH323WRAPLIBDIR); ln -s liboh323wrap.so liboh323wrap.so.1
+	cd $(DESTDIR)$(OH323WRAPLIBDIR); ln -s liboh323wrap.so liboh323wrap.so.1.1
 endif
 
 clean:
--- ./asterisk-driver/Makefile.org	Thu Jul  4 12:52:16 2002
+++ ./asterisk-driver/Makefile	Fri May  9 14:45:43 2003
@@ -32,9 +32,15 @@
 all: chan_oh323.so
 
 install:
-	$(INSTALL) chan_oh323.so $(ASTERISKMODDIR)
-	@if [ ! -f $(ASTERISKETCDIR)/oh323.conf ]; then \
-		$(INSTALL) oh323.conf.sample $(ASTERISKETCDIR)/oh323.conf; \
+	if [ ! -d $(DESTDIR)$(ASTERISKMODDIR) ]; then \
+		$(INSTALL) -d $(DESTDIR)$(ASTERISKMODDIR) \
+	fi
+	$(INSTALL) chan_oh323.so $(DESTDIR)$(ASTERISKMODDIR)
+	if [ ! -d $(DESTDIR)$(ASTERISKETCDIR) ]; then \
+		$(INSTALL) -d $(DESTDIR)$(ASTERISKETCDIR) \
+	fi
+	@if [ ! -f $(DESTDIR)$(ASTERISKETCDIR)/oh323.conf ]; then \
+		$(INSTALL) oh323.conf.sample $(DESTDIR)$(ASTERISKETCDIR)/oh323.conf; \
 	fi
 
 clean:
--- ./Makefile.org	Wed Aug 28 17:10:42 2002
+++ ./Makefile	Fri May  9 14:22:27 2003
@@ -108,12 +108,12 @@
 INSTALL=install
 TOUCH=touch
 AR=ar
-OH323WRAPINCLUDE=-I/usr/include \
-				 -I/usr/include/crypto \
-				 -I$(PWLIBDIR)/include/ptlib/unix \
-				 -I$(PWLIBDIR)/include \
-				 -I$(OPENH323DIR)/include \
-				 -I../asterisk-driver
+OH323WRAPINCLUDE=-I/usr/include/crypto \
+		 -I$(PWLIBDIR)/include/ptlib/unix \
+		 -I$(PWLIBDIR)/include \
+		 -I$(OPENH323DIR)/include \
+		 -I../asterisk-driver \
+		 -I/usr/include
 CPPFLAGS=-Wall -mcpu=i586 $(PWLIBFLAGS) $(OPENH323FLAGS)
 ASTERISKINCLUDE=-I$(ASTERISKSRCDIR)/include -I../wrapper
 CFLAGS=-Wall -mcpu=i586
